pyproject.toml

```diff
@@ -222,6 +222,10 @@ extend-ignore = [
     # pep8-naming (N)
     # NOTE: some of these can/should be fixed, but this changes the API.
     "N8",
+    "N999",  # Invalid module name                     # TODO: fix
+
+    # NumPy-specific rules (NPY)
+    "NPY002", # Replace legacy `np.random.rand` call with `np.random.Generator`
```

Comment:
Huh... I thought we modernized the `np.random` usage?

Reply:
More issues found

Reply:
😢